makelib.sh

```diff
@@ -0,0 +1,4 @@
+main="$1"; shift
```

Comment:
"main" isn't really needed, one could use $1 at the 2 places where it matters below and just $* for the AR line.

Reply:
I replicated the previous scripts directly, I prefer as you suggest.